tjmonopix2/system/cmd.py

```diff
@@ -9,8 +9,13 @@
 
 
 class cmd(RegisterHardwareLayer):
-    '''Implement master RD53 configuration and timing interface driver.
-    '''
+    """Implement master RD53 configuration and timing interface driver.
+
+    Parameters
+    ----------
+    RegisterHardwareLayer : class
+        Allows register read write operations
+    """
```

Comment:
Same as for the `DAQBoard` driver. The naming is a bit misleading and copied from the bdaq53 project. I would refer to it as:
Driver for the command encoder firmware module.